src/assembler/assembler.jl

```diff
@@ -456,6 +456,20 @@ function _count_n_elts(
     return n
 end
 
+"""
```

Comment:
Can you please merge the three `_count_n_elts` doc strings into a unique one, located at the top of the first method declaration? Take a look at `src/fespace/fespace.jl` -> `TrialFESpace` constructor for inspiration.